authors/apps/articles/views.py

```diff
@@ -24,6 +27,9 @@ class ArticleAPIView(generics.ListCreateAPIView):
     queryset = Article.objects.all()
     serializer_class = ArticleSerializer
     renderer_classes = (ArticleJSONRenderer,)
+    permission_classes = (AllowAny,)
```

Comment:
I thought view all articles was meant to be for anyone since according to Daniel the PO, there was no difference between subscribed users and anonymous users